packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1097,7 +1097,7 @@ describe("calcite-combobox", () => {
       await page.waitForChanges();
       await page.keyboard.press("Tab");
       await page.waitForChanges();
-      expect(await combobox.getProperty("value")).toBe("");
+      expect(await inputEl.getProperty("value")).toBe("");
```

Comment:
Great catch, but can we submit these renames separately? There are no other changes in this file related to the PR, so it's out of scope.

Reply:
These changes are related i think. This helps in asserting the input clear on blur functionality via keyboard & Mouse. Though these tests were added earlier as part of https://github.com/Esri/calcite-design-system/pull/7721 , they couldn't identify regression when https://github.com/Esri/calcite-design-system/pull/7954 is installed . Made changes to this test so that they can detect any regressions from now on.